public/src/components/teacher/Results.tsx

```diff
@@ -20,7 +20,7 @@ class Results extends React.Component<IResultsProp, IResultsState> {
         if (this.props.students[0] && this.props.students[0].course.assignments[0]) {
```

Comment:
Is it safe to use `0` here?

Reply:
Wouldn't it be better to replace common phrases such as `this.props.student[0]` with a new variable `student` or `stud` or `currentStudent` or `currStud`?